examples/workflow-theia/src/browser/workflow-glsp-client-contribution.ts

```diff
@@ -34,4 +38,15 @@ export class WorkflowGLSPClientContribution extends BaseGLSPClientContribution {
             ['message']: 'Custom Options Available'
         };
     }
+
+    protected override async createGLSPClient(connectionProvider: ConnectionProvider): Promise<GLSPClient> {
+        const envVar = await this.envVariableServer.getValue('WEBSOCKET_PORT');
+        const webSocketPort = envVar && envVar.value ? Number.parseInt(envVar.value, 10) : undefined;
```

Comment:
We should probably do some error guarding here.
If the websocket envar is defined we should check that its a valid number  (so not NaN or `undefined`) and throw an Error if that's not the case.